flow/designs/asap7/mock-array-big/Element/io.tcl

```diff
@@ -51,4 +51,4 @@ foreach {direction direction2 names} $assignments {
     set_io_pin_constraint -mirrored_pins $mirrored
 }
 
-set_io_pin_constraint -region top:* -pin_names clock
+set_io_pin_constraint -region top:{0 0 3 0} -pin_names clock
```

Comment:
@nerriav 
Can you check with below format?
```
set_io_pin_constraint -direction input -region up:{0 0 3 0} -pin_names clock
```

Reply:
same error massage ):

Reply:
@maliberty 
Can you check this?

Reply:
Refer this [example](https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/ppl/test/add_constraint8.tcl).
```
set_io_pin_constraint -region top:0-3 -pin_names clock
```

Reply:
Thanks !!!. it worked.
@eder-matheus 
documentation is a little bit confusing here , probably need an update
https://github.com/The-OpenROAD-Project/OpenROAD/blob/86d1769f8d658dd25ee55c1adc5405d4e76c5e97/src/ppl/README.md?plain=1#L109

Reply:
@nerriav 
Try above constraint, it resolve your issue and place pin withing 0-3um region.
in log `[INFO PPL-0048] Restrict pins [clock] to region 0.0u-3.0u at the top edge.`
![Screenshot from 2023-05-05 17-30-05](https://user-images.githubusercontent.com/85787913/236452509-558d7cdb-2965-42a1-afe0-a5b0da327ec5.png)

Reply:
@nerriav The `-region up` is used only when placing the pins in the grid created with `define_pin_shape_pattern `, and it's not for the boundaries of the edge. I'll modify the documentation to make it clear.